include/net/socket.h

```diff
@@ -688,8 +688,23 @@ static inline ssize_t recv(int sock, void *buf, size_t max_len, int flags)
 	return zsock_recv(sock, buf, max_len, flags);
```

Comment:
Minor nit, please change the commit subject to `net: socket: ....` as the prefix should not be a path but describe the logical component that is being changed

Reply:
Done.